source/core/rtc_adapter/VideoSendAdapter.cc

```diff
@@ -198,6 +198,8 @@ bool VideoSendAdapterImpl::init()
         m_rtpRtcp->SetMid(mid);
     }
 
+    m_rtpRtcp->SetMaxRtpPacketSize(1200);
```

Comment:
don't use direct value for those settings.  add:
static constexpr int kMaxRtpPacketSize = 1200;  
in the file.